ui/app/routes/variables/path.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+
+export default class VariablesPathRoute extends Route {
+  model({ absolutePath }) {
+    let treeAtPath = this.modelFor('variables').pathTree.findPath(absolutePath);
```

Comment:
Originally written to let us do recursive tree traversal and can still be expanded to allow for that (we could have `<PathTree>` call itself with its children).